python/PlotOptions.py

```diff
@@ -0,0 +1,51 @@
+from optparse import OptionParser
+
+parser=OptionParser()
+
+parser.add_option('-f','--file',action="append",dest='file',type=str)
+parser.add_option('--OutputName',action="store",dest="OutputName",type=str,default="Myconfig",help="Set Output file name, default is: Myconfig")
```

Comment:
change the default here to be more descriptive and use camelCase, e.g.:

```
plotConfig.cfg
```

Reply:
done